bitnami/kafka/Chart.yaml

```
@@ -29,4 +29,4 @@ name: kafka
 sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/kafka
   - https://kafka.apache.org/
-version: 18.0.8
+version: 18.0.9
```

You are adding a new feature, according to [semver](https://semver.org/) the version should be:
```suggestion
version: 18.1.0
```